tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -509,14 +508,14 @@ public:
    }
 
    // if the fObjects vector type is derived from TObject, return the name of the object
-   template <typename T = HIST, typename std::enable_if<std::is_base_of<TObject, T>::value>::type * = nullptr>
+   template <typename T = HIST, std::enable_if_t<std::is_base_of<TObject, T>::value, int> = 0>
```

Comment:
shouldn't that be `std::enable_if_t<std::is_base_of<TObject, T>::value>* = nullptr`?

Reply:
it should make no difference, this form takes less characters and is the one used everywhere else

Reply:
> there is no template type or non-type argument there

mmh not quite, the template parameter is of type `int` there. I can explain in person :)